format/test/format_test.bats

```diff
@@ -6,151 +6,130 @@ bats_load_library "bats-assert"
 
 # No arguments: will use git ls-files
 @test "should run prettier on javascript using git ls-files" {
-    run bazel run //format/test:format_javascript
+    run bazel run //format/test:format_JavaScript_with_prettier
     assert_success
 
-    assert_output --partial "Formatting JavaScript with Prettier..."
```

Comment:
and this wasn't replaced with something equivalent?

Reply:
No, because this test doesn't execute the `multirun` which prints `Running @@//tools/format:format_Markdown_with_prettier`. Instead it calls that one binary directly.

Reply:
Oh and look, Keith is helping us out :)
https://github.com/keith/rules_multirun/pull/35